internal/controllers/core/cmd/controller.go

```diff
@@ -557,19 +568,6 @@ func indexCmd(obj client.Object) []indexer.Key {
 	return result
 }
 
-func (r *Controller) CreateBuilder(mgr ctrl.Manager) (*builder.Builder, error) {
```

Comment:
(I moved this to the top of the file so it's next to the constructor like the others for consistency/ease of finding it)